src/Microsoft.Diagnostics.Monitoring.RestServer/CorsConfiguration.cs

```diff
@@ -11,6 +13,8 @@ namespace Microsoft.Diagnostics.Monitoring.RestServer
 {
     public class CorsConfiguration
     {
+        [Description("List of allowed cors origins, separated by ;")]
```

Comment:
nit: CORS should be capitalized since it is an acronym. Maybe "List of allowed CORS origins, separated by semicolons." to make it a complete punctuated sentence.